app/models/book.rb

```diff
@@ -2,6 +2,12 @@ class Book < ApplicationRecord
   include MultiPageTool
   include Featureable
 
+  NON_INDEX_BOOK_SLUGS = %w[
+    days-of-war-nights-of-love-ne-plus-ultra-edition
+  ].map(&:freeze).freeze
+
+  scope :for_index, -> { where.not(slug: NON_INDEX_BOOK_SLUGS).reorder(position: :asc).order(title: :asc) }
```

Comment:
And if you wanna remove, you could add another bit on Book like “show_on_index” and remove the hard coded constant to remove the _Ne Plus edition_.